rali/rali_pybind/amd/rali/plugin/pytorch.py

```diff
@@ -99,6 +99,9 @@ def __iter__(self):
     def __len__(self):
         return self.len
 
+    def __del__(self):
```

Comment:
Codacy found an issue: [Missing docstring in magic method (D105)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5745979)